libraries/abstractions/transport/secure_sockets/transport_secure_sockets.h

```diff
@@ -0,0 +1,188 @@
+/*
+ * Copyright (C) 2020 Amazon.com, Inc. or its affiliates.  All Rights Reserved.
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy of
+ * this software and associated documentation files (the "Software"), to deal in
+ * the Software without restriction, including without limitation the rights to
+ * use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of
+ * the Software, and to permit persons to whom the Software is furnished to do so,
+ * subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in all
+ * copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS
+ * FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR
+ * COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER
+ * IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN
+ * CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+ *
+ * http://aws.amazon.com/freertos
+ * http://www.FreeRTOS.org
+ */
+
+/**
+ * @file transport_secure_sockets.h
+ * @brief Secure Sockets based API for transport interface implementation.
+ */
+
+#ifndef TRANSPORT_SECURE_SOCKETS_H
+#define TRANSPORT_SECURE_SOCKETS_H
+
+/* standard includes. */
+#include <stdbool.h>
```

Comment:
We are avoiding use of `stdbool.h` inclusion to be C90 compliant. You can use the definition of `bool` used in MQTT [here](https://github.com/FreeRTOS/coreMQTT/blob/master/source/include/core_mqtt_serializer.h#L42-L48) instead.

Reply:
fixed in 9e9a7389c3e1f06de838bfe2cd3e5254c1cc8bb1